app/views/search/all_content.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= render :partial => "search/nav", :locals => { :section => "search-all" } %>
+
+<div class="container">
+    <% if  @nodes.all?{|k,v| v.blank?} %>
+        <h4>
+            No Content found for <%= params[:query] %> :-(
+            </br></br>
+            Try searching <a href="/search">for another topic</a>
+        </h4>
+    <% else %>
+
+        <h2> Results for <%= params[:query] %></h2>
+        <% unless @nodes[:notes].blank? %>
+            <h3> Notes </h3>
+            <%= render partial: "notes/notes", locals: { notes: @nodes[:notes] } %>
+        <% end%>
+        <% unless  @nodes[:wiki].blank? %>
+            <h3> Wikis </h3>
+            <%= render partial: "wiki/wikis", locals: { wiki: @nodes[:wikis] } %>
```

Comment:
Hi @cesswairimu !
What do you think of this ?  
In this file  : https://github.com/publiclab/plots2/blob/master/app/views/wiki/_wikis.html.erb , we are not using the local variable **wiki**  to display the result  . 
Hence we should add this line : 
`<% wikis = wikis || @wikis %> # accept local if present, default to instance` there . 
and then use **wikis** variable everywhere .